java-frontend/src/main/java/org/sonar/java/ast/visitors/SyntaxHighlighterVisitor.java

```diff
@@ -43,15 +45,18 @@
   private final SonarComponents sonarComponents;
   private final Map<Tree.Kind, TypeOfText> typesByKind;
   private final Set<String> keywords;
+  private final Set<String> restrictedKeywords;
+  private static final String INTERFACE_KEYWORD_VALUE = JavaKeyword.INTERFACE.getValue();
```

Comment:
I found only one usage of this constant, imo `JavaKeyword.INTERFACE` could be used directly.